ScoutSuite/providers/gcp/rules/findings/computeengine-vpc-flow-logs-disabled.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "VPC Flow Logs Not Enabled",
+    "rationale": "VPC Flow Logs were not enabled for this subnet. It is best practice to enable Flow Logs to some degree in order to have network visibility in the event of resource compromise, as well as source data for threat detections.",
+    "dashboard_name": "Subnetwork",
```

Comment:
Can you add the compliance and references section for this rule? It would look like:
"compliance": [
        {
            "name": "CIS Google Cloud Platform Foundations",
            "version": "1.1.0",
            "reference": "..."
        }
    ],
    "references": [
          "..."
    ],

Reply:
thanks for feedback! ill take a crack tn.